src/services/productProcessor.ts

```
@@ -0,0 +1,72 @@
+      availableForPickup: rawProduct.available_for_pickup === 'true',
+      colors: safeJsonParse(rawProduct.colors, []),
+      sizes: safeJsonParse(rawProduct.sizes, []),
+      tags: safeJsonParse(rawProduct.tags, []),
+      url: safeString(rawProduct.url, '')
+    };
+  }
+
+  static processProducts(rawProducts: WalmartProduct[]): ProcessedProduct[] {
+    return rawProducts.map(product => this.processProduct(product));
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)